### PR TITLE
AGTS 2019: Quarterfinals & GTS Reorganisation

### PR DESCRIPTION
**Pull request by fajar13k**
- Start of Advanced Global Taiko Showdown 2019 article
- Reorganisation of Global Taiko Showdown series

---

### CODE REVIEW

**Comment by cl8n** [comment]
should be
```
GTS
├── IGTS_2019
├── AGTS_2019
└── ...
```
to avoid having pages with no parents

File: wiki/Tournaments/GTS/AGTS_2019/en.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+| Semifinals | 2019-08-31/2019-09-01 |
+| Finals | 2019-09-07/2019-09-08 |
+| Grand Finals | 2019-09-14/2019-09-15 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, exclusive team banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | exclusive team banner |
```
Comment (cl8n) [inline wiki/Tournaments/GTS/AGTS_2019/en.md]:
```suggestion
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Exclusive team banner |
```

File: wiki/Tournaments/GTS/AGTS_2019/en.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | exclusive team banner |
```
Comment (cl8n) [inline wiki/Tournaments/GTS/AGTS_2019/en.md]:
```suggestion
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Exclusive team banner |
```

File: wiki/Tournaments/GTS/AGTS_2019/en.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+| Designer | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
+| Statisticians | ![][flag_DE] [IceDynamix](https://osu.ppy.sh/users/8599070), ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_CA] [Leadenginger](https://osu.ppy.sh/users/3799991), ![][flag_MY] [Minisora](https://osu.ppy.sh/users/9627666) |
+| Translators | ![][flag_BR] [AnonX32](https://osu.ppy.sh/users/2730270), ![][flag_KR] [BamgoeSN](https://osu.ppy.sh/users/1152851), ![][flag_DE] [Capu](https://osu.ppy.sh/users/2474015), ![][flag_HK] [Cynplytholowazy](https://osu.ppy.sh/users/3901754), ![][flag_PL] [Deroo](https://osu.ppy.sh/users/8360475), ![][flag_EC] [Gamelan4](https://osu.ppy.sh/users/9856910), ![][flag_TW] [InfinityAstral](https://osu.ppy.sh/users/9913256), ![][flag_US] [jyake](https://osu.ppy.sh/users/9099822), ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
+| Wiki | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
```
Comment (cl8n) [inline wiki/Tournaments/GTS/AGTS_2019/en.md]:
```suggestion
| Wiki Editor | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
```
is this even worth listing tho

File: wiki/Tournaments/GTS/AGTS_2019/en.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | exclusive team banner |
+
+## Organisation
+
+The Advanced Global Taiko Showdown 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
+| Map Selectors | ![][flag_DE] [Nepuri](https://osu.ppy.sh/users/6637817), ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097) |
```
Comment (cl8n) [inline wiki/Tournaments/GTS/AGTS_2019/en.md]: these roles should be written singular (so "Map Selector", "Referee", etc)

Comment (fajar13k) [reply wiki/Tournaments/GTS/AGTS_2019/en.md]: New ASC?

Comment (cl8n) [reply wiki/Tournaments/GTS/AGTS_2019/en.md]: it's just grammar. you have the singular "Position" as the header, and "Map Selectors" for example isn't a single position/role

your options here would be either "Map Selector" or "Map Selection". I think the first is more like a "position" and the second is more like a "role"... subtle distinction. if u go with the first then ig u should change "Management" to "Manager" as well

File: wiki/Tournaments/GTS/AGTS_2019/en.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+
+**[Download the map pack here! (83 MB)](https://mega.nz/#!spkR0agQ!MokudfOpqItg9GU4KnYaD-jRjMLVYctMnBeRmWDtGDA)**
+
+- NoMod
+  - [Function Phantom - Euclid (salchow) \[Green's Fifth Axiom\]](https://osu.ppy.sh/beatmapsets/941145#taiko/2002626)
+  - [S.S.H. - BassTest (Ak1o) \[TEST\]](https://osu.ppy.sh/beatmapsets/666764#taiko/1411059)
+  - [Chroma - I (rubies87) \[Oni\]](https://osu.ppy.sh/beatmapsets/883570#taiko/1933318)
+  - [Camellia - \*Feels Seasickness...\* (Genjuro) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/1014457#taiko/2126517)
+  - [Sadkey - Further (Airuei Redo) (Nifty) \[Discovery\]](https://osu.ppy.sh/beatmapsets/741764#taiko/1564820)
+- EX
```
Comment (cl8n) [inline wiki/Tournaments/GTS/AGTS_2019/en.md]: explain in parenthesis what this means, the only mention of "EX" further down (in an unrelated section of the rules) says that EX is for harder maps but I still don't know what mods are allowed

**Comment by fajar13k** [comment]
Preferably going to close this for now, and reorganise it might be better in the end of tournament. (approx. 14/09)

**Pull request closed without merging**